rosidl_cmake/cmake/rosidl_cmake_export_typesupport_libraries-extras.cmake.in

```diff
@@ -0,0 +1,44 @@
+# generated from rosidl_cmake/cmake/template/rosidl_cmake_export_typesupport_libraries.cmake.in
```

Comment:
Copyright?

Reply:
Neither of the other `.cmake.in` templates have copyright notes. I think that was mostly so that generated code embedded into other packages doesn't have implicitly the same copyright (e.g. if that package wants to choose a different license).